src/modules/ie-objects/services/ie-objects.service.spec.ts

```diff
@@ -269,6 +271,36 @@ describe('ieObjectsService', () => {
 		});
 	});
 
+	describe('findObjectsForSitemap', () => {
+		it('should throw an error when it fails to get object', async () => {
+			mockDataService.execute.mockResolvedValueOnce('');
+			try {
+				await ieObjectsService.findIeObjectsForSitemap(
+					[IeObjectLicense.PUBLIEK_METADATA_LTD, IeObjectLicense.PUBLIEK_METADATA_ALL],
+					0,
+					50
+				);
```

Comment:
add a fail statement after this await, since we want the test to fail if the function does not throw an exception